### PR TITLE
WIP: Better test

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: test/test_util.dart [hunk 1]
```diff
@@ -7,10 +7,10 @@ import 'dart:async';
 import 'package:shelf/shelf.dart';
 
 // "hello,"
-const HELLO_BYTES = const [104, 101, 108, 108, 111, 44];
+const helloBytes = const [104, 101, 108, 108, 111, 44];
```
Comment (devoncarew) [inline test/test_util.dart]: Curious, has the style guide recommendation changed? I know flutter uses `kMyConstantName`; I thought regular dart was still `MY_CONSTANT_NAME`.

Comment (devoncarew) [inline test/test_util.dart]: WHICH IS ACTUALLY PRETTY LOUD, So I'm happy to have camel case.

Comment (kevmoo) [inline test/test_util.dart]: https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names 😄

File: analysis_options.yaml [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+    - prefer_final_fields
+    - prefer_is_not_empty
+    - slash_for_doc_comments
+    - super_goes_last
+    - test_types_in_equals
+    - throw_in_finally
+    - type_init_formals
+    - unnecessary_brace_in_string_interps
+    - unrelated_type_equality_checks
+    - valid_regexps
```
Comment (nex3) [inline analysis_options.yaml]: I don't want to rehash the discussion on https://github.com/dart-lang/package_resolver/pull/6, but please remove these lints.

**Pull request merged**